src/Definition/MySQL.php

```diff
@@ -69,6 +70,48 @@ public function toArray()
         ];
     }
 
+    /**
+     * @param array $options
+     * @return string|null
+     * @throws InvalidOptionException
+     */
+    private function processLength($options)
+    {
+        if (!isset($options['length'])) {
+            return null;
+        }
+        if (is_array($options['length'])) {
+            return $this->getLengthFromArray($options['length']);
+        }
+        return $options['length'];
+    }
+
+    /**
+     * @param array $lengthOptions
+     * @throws InvalidOptionException
+     * @return null|string
+     */
+    private function getLengthFromArray($lengthOptions)
+    {
+        $expectedOptions = ['character_maximum', 'numeric_precision', 'numeric_scale'];
+        $diff = array_diff(array_keys($lengthOptions), $expectedOptions);
+        if (count($diff) > 0) {
+            throw new InvalidOptionException(sprintf('Length option "%s" not supported', $diff[0]));
+        }
+
+        $characterMaximum = isset($lengthOptions['character_maximum']) ? $lengthOptions['character_maximum'] : null;
```

Comment:
MySQL vracia stlpec s nazvom `character_maximum_length` a nie `character_maximum` ... tak si to bude musel programator premapovat .... ale asi to teda chceme mat vsade rovnako, OK (pytam sa pre istotu).

https://dev.mysql.com/doc/refman/8.0/en/char.html#:~:text=The%20CHAR%20and%20VARCHAR%20types%20are%20declared%20with%20a%20length,value%20from%200%20to%20255.

Reply:
vím - dal jsem to ale všude stejný